spock-core/src/main/java/org/spockframework/tempdir/TempDirConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package org.spockframework.tempdir;
+
+import org.spockframework.util.Beta;
+import spock.config.ConfigurationObject;
+
+import java.nio.file.Path;
+
+/**
+ *
+ * Configuration settings for the TempDir extension.
+ *
+ * <p>Example:
+ * <pre>
+ * tempdir {
+ *   baseDir Paths.get("/tmp) // java.nio.Path object, default null, which means system property "java.io.tmpdir"
```

Comment:
Sorry, but now it's invalid, you miss the closing quote. :-)
The line ist still very long, but at least it is better now if you also change it in the docs.
Can we keep it like this now if the quote is added,
or is it still too long, what do you think @leonard84?